its/core-it-support/core-it-plugins/maven-it-plugin-uses-wagon/src/main/java/org/apache/maven/plugin/coreit/LookupWagonMojo.java

```diff
@@ -92,21 +92,11 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
         getLog().info("[MAVEN-CORE-IT-LOG] Creating output file " + outputFile);
 
-        OutputStream out = null;
-        try {
-            outputFile.getParentFile().mkdirs();
```

Comment:
without this its not possible.  This confuses the code as has nothing to to with the try catch.

When looking into this argument relativized as used in fluid API melding down boilerplate even more

- https://github.com/apache/maven/pull/2395

10 LOC bare boilerplate less.

lees seems more. 

stay lean to get clean.

@elharo @gnodet